node/pod.go

```diff
@@ -390,3 +390,15 @@ func (pc *PodController) deletePodHandler(ctx context.Context, key string) (retE
 	}
 	return nil
 }
+
+// FilterPodsForNodeName creates an event filter function that filters pod events such that pod.Sepc.NodeName matches the provided name
+// Use the return value of this as the PodEventFilterFunc in PodControllerConfig
+func FilterPodsForNodeName(name string) func(interface{}) bool {
```

Comment:
This to me almost seems more of a "utils" or "helpers" thing? Also, it'd be neat if these were written in such a way that filters can be composed -- maybe if .filterfunc was an array?

Reply:
I was thinking something like a function that bundles them together... so like `Filter(filters ...FilterFunc) FilterFunc`

Reply:
Moved to nodeutils.
Also added a function to take a list of filters and make one filter.